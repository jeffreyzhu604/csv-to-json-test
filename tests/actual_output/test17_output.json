[
[,
{"fname":"jeff","lname":"zhu","age":90,"countryOfResidences":"          \\\\canada     "},,
{"fname":"hojun","lname":"","age":"\"26\"","countryOfResidences":"korea"},,
]
]
