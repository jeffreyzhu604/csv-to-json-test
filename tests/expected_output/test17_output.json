unknown parameter ../tests/test_files/test17_check_col_error.csv.
[
]
]
